ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/GoogleCloudSccIntegrationForm.tsx

```diff
@@ -14,11 +14,13 @@ import { IntegrationFormProps } from '../integrationFormTypes';
 
 import IntegrationFormActions from '../IntegrationFormActions';
 import FormLabelGroup from '../FormLabelGroup';
+import useFeatureFlags from '../../../../hooks/useFeatureFlags';
 
 export type GoogleCloudSccIntegration = {
     cscc: {
         serviceAccount: string;
         sourceId: string;
+        wifEnabled: boolean;
```

Comment:
Looking forward to day when we remove this redundant type declaration, can you also in frontend file which corresponds to backend source of truth:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/types/notifier.proto.ts#L55-L58